tools/docker/scripts/lsblk.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/sh
+# tiny lsblk(8) equivalent to make integration tests distro-agnostic
+set -eu
+
+echo 'NAME MAJ:MIN RM      SIZE RO | MAGIC'
+
+for name in $(ls /sys/block | grep -v loop)
```

Comment:
nit: Is it possible to avoid grep? Grep is also pretty universal, so if there's no other straightforward way that's alright, but it's technically not a coreutil so it would be nice to avoid

Reply:
Hmm, microVMs on my Debian EC2 box doesn't have loopX but it has on Buildkite...

https://buildkite.com/firecracker-microvm/firecracker-containerd/builds/653#846f0b05-81a2-441f-82fd-0f95b124fb5f

Reply:
Used case...esac instead.